packages/frontend/src/components/Files/File.tsx

```diff
@@ -42,13 +42,24 @@ export const NodeItem = ({ node, row }: any) => {
     <Grid
       container
       spacing={2}
-      sx={{ paddingBottom: '18px', backgroundColor: row % 2 ? Colors.lightgrey : Colors.white }}
+      sx={{
+        paddingBottom: '18px',
+        backgroundColor: row % 2 ? Colors.lightgrey : Colors.white,
+        cursor: 'pointer',
+        textDecoration: 'none',
+      }}
+      component="a"
+      target="_blank"
+      rel="noopener noreferrer"
+      href={`${process.env.REACT_APP_ALFRESCO_API_URL}/alfresco/versions/1/nodes/${id}/content?attachment=false`}
```

Comment:
I've been told that it's better to have process.env-calls in a const at the same level as that NodeTypes in this file (or "higher"), as each process.env call is a comparatively slow process, at least if done a lot. So that's what I'll keep on parroting! 😄